Source/Views/TextViews/SPTextView.m

```diff
@@ -836,7 +836,7 @@ - (void)doCompletionByUsingSpellChecker:(BOOL)isDictMode fuzzyMode:(BOOL)fuzzySe
 			NSArray *allTables = [[dbs objectForKey:conID] allKeys];
 			// Check if found table name is known, if not parse for aliases
 			if(![allTables containsObject:[NSString stringWithFormat:@"%@%@%@", conID, SPUniqueSchemaDelimiter, tableName]]) {
-				NSString *currentQuery = [[self string] substringWithRange:[customQueryInstance currentQueryRange]];
+                NSString *currentQuery = [[self string] substringWithRange:NSMakeRange(0, self.string.length)];
```

Comment:
`self.string substringWithRange:Range of whole string` - why don't we take the string itself? 😄